sdk/communication/azure-communication-callingserver/pom.xml

```diff
@@ -24,11 +24,54 @@
   </description>
   <url>https://github.com/Azure/azure-sdk-for-java</url>
 
+  <repositories>
+<!--    <repository>-->
+<!--      <id>azure-sdk-for-java-communication</id>-->
+<!--      <url>https://pkgs.dev.azure.com/azure-sdk/internal/_packaging/azure-sdk-for-java-communication/maven/v1</url>-->
+<!--      <releases>-->
+<!--        <enabled>true</enabled>-->
+<!--      </releases>-->
+<!--      <snapshots>-->
+<!--        <enabled>true</enabled>-->
+<!--      </snapshots>-->
+<!--    </repository>-->
+    <repository>
+      <id>azure-sdk-for-java-communication-interaction</id>
```

Comment:
should this be named callautomation?

Reply:
This is for the artifact feed that I used for sharing internally to test. The name of that was determined as "interaction". This will be removed once we set up the nightly build feed